File: Python Excel Examples/WorksheetsApiDemo/comments/getComments.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration

appId = "your id"
appKey = "your key"
baseUrl="https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.worksheets_api.WorksheetsApi(configuration)

name = "GetComments.xlsx"
storage = ""
folder = "ExcelDocument"
sheet_name = "Sheet1"
result = api.get_comments(name, sheet_name, storage=storage, folder=folder)

File: PHP Excel Examples/WorksheetsApiDemo/Comments/getComments.php
<?php

use Spire\Cloud\Excel\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey, $baseUrl);
$apiInstance = new WorksheetsApi($configuration);

$name = "GetComments.xlsx";
$storage = null;
$folder = "input";
$sheetName = "Sheet2";
$result = $apiInstance->getComments($name, $sheetName,$folder, $storage);